src/components/Main/Map.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useRef, useEffect, useState } from "react";
+import mapboxgl from "!mapbox-gl"; // eslint-disable-line import/no-webpack-loader-syntax
+import { fetchCityAndPlaces } from "../../api/fetchCityAndPlaces";
+import style from "./Map.module.css";
+
+export function Map() {
+  mapboxgl.accessToken = process.env.REACT_APP_MAPBOX_ACCESS_TOKEN;
+
+  const mapContainer = useRef(null);
+  const map = useRef(null);
+  const [lng, setLat] = useState(-0.1276);
+  const [lat, setLng] = useState(51.5072); // default [-0.1276, 51.5072] is London
+  const [zoom, setZoom] = useState(13);
+  const [places, setPlaces] = useState([]);
+
+  // mock data, will get them from user input later
+  const searchWord = "cafe";
+  const proximity = `${lng},${lat}`;
+  const navProfile = "walking";
+
+  // Fetch the map based on the longitude and latitude and show it on the screen.
+  useEffect(() => {
```

Comment:
I don't think you are 'fetching'  the map. You are creating it from the lib

Reply:
how this works? I mean I thought they store all data somewhere and I call the API to get what I want specifically? I cannt ask them to generate something for me?